Changes

```diff
@@ -2,7 +2,10 @@ Revision History for ConsumerGroups
 
 {{$NEXT}}
 
-0.05      2023-11-07 07:54:26+00:00 UTC
+0.06    2024-03-06 14:45:00+00:00 UTC
```

Comment:
remove these 2 lines after `{{NEXT}}`. dzil will do that automatically when run `dzil release`